data/default_imsim_configs

```diff
@@ -14,6 +15,7 @@ catalog = default
 
 [checkpointing]
 nobj = 200
+cleanup = True
```

Comment:
Does Adrian needs this default nobj larger now?

Reply:
He might, but for running on a KNL node, I think this is still basically the right number.